Add information about quota support (#467)
iterate-ch/docs: mountainduck/connect/index.md

| **[Recent Files](sync.md#recent-files)** | –                                                                     | Available in the status bar (macOS) or taskbar (Windows)                      |
| **[Lock Files](../locking.md)**          | ✔︎                                                                    | ✔                                                                             |
| **[Share Files](../share.md)**           | ✔                                                                     | ✔                                                                             |

## Quota support

> Mountain Duck has support for the overall quota of a server but quota reporting is not supported for all protocols.

Reviewer: ```suggestion
Mountain Duck display the available overall quota available on the server as available disk space on the mounted volume.
```

| Protocol                 | Support |
|--------------------------| :---: |
| Local Disk               | ✅ |
| SFTP                     | ✅ |
| FTP                      | ❌ |
| WebDAV			       | ✅ |
| Google Drive             | ✅ |
| Google Cloud Storage     | ❌ |
| Microsoft OneDrive       | ✅ |
| Microsoft Sharepoint     | ✅ |
| Files.com                | ❌ |
| DRACOON                  | ✅ |
| Backblaze B2             | ❌ |
| Box.com                  | ❌ |
| Dropbox                  | ✅ |
| Nextcloud                | ✅ |
| ownCloud                 | ✅ |
| S3                       | ❌ |
| SMB                      | ✅ |
| Windows Azure            | ❌ |

> | OpenStack Object Storage | ✅ |

Reviewer: May want to link to protocol pages.

```{admonition} Limited Support
:class: attention

> - **[Microsoft OneDrive](../../protocols/onedrive.md#quota)**: Quota is only supported if a folder other than the virtual root is mounted.

Reviewer: ```suggestion
- **[Microsoft OneDrive](../../protocols/onedrive.md#quota)**: Quota is only supported when setting the _Path_ in the bookmark configuration to a folder different from `/`.
```

> - **[Microsoft SharePoint](../../protocols/sharepoint.md#quota)**: Quota is only supported for the *Drives* folders within the respective SharePoint site folders.

Reviewer: ```suggestion
- **[Microsoft SharePoint](../../protocols/sharepoint.md#quota)**: Quota is only supported when setting the _Path_ in the bookmark configuration to a *Drives* folders in a SharePoint site folder.
```

```

> Mountain Duck can't display the correct remaining free space for protocols that can't report quota. Finder and Windows Explorer will show exabyte values within the *Online* mode and the available space within the synchronization cache location within the *Smart Synchronization* mode for servers and protocols that don't have support for quota reporting.

Reviewer: ```suggestion
Some protocols do not report available quota. Finder and Windows Explorer will show exabyte values in *Online* connect mode and the available space within the synchronization cache location on your local disk in *Smart Synchronization* connect mode.
```
